src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -708,6 +708,11 @@ public synchronized void updateMainTrackWithRecentlyFinishedBroadcast(Broadcast
 
 				if (mainBroadcast.isZombi()) {
 					logger.info("Deleting main track streamId:{} because it's a zombi stream and there is no activeSubtrack", mainBroadcast.getStreamId());
+					// delete all subtracks which are not broadcasting
+					long size = getDataStore().getSubtrackCount(mainBroadcast.getStreamId(), null, null);
```

Comment:
I don't think it's a good idea to get all subtrack counts and delete all of them. 

Two reminders
1. There can be other applications than video conferencing that can use subtracks 
2. The subtracks may not be zombi streams, so they should not be deleted.

Reply:
Thanks for your feedback. I will reconsider the design.

Reply:
I added a check if the stream is zombi or not. I believe this is will be enough. Can you please review it? @mekya

Reply:
isZombi stuff is ok. 

On the other hand, I don't want to trust this code to remove the subtracks.  It seems to me an extra database operation which creates overhead and also seems like a workaround solution.

What is the blocker to remove the subtracks in their own pipeline if they are zombi?